products/accesscontextmanager/terraform.yaml

```diff
@@ -14,6 +14,7 @@
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
   AccessPolicy: !ruby/object:Overrides::Terraform::ResourceOverride
+    supports_indirect_user_project_override: true
```

Comment:
As above- this can be removed (across all 4 ACM resources) as we want this to work for all generated resources.